intermediate_source/forward_ad_usage.py

```diff
@@ -172,5 +172,75 @@ def jvp(ctx, gI):
                          check_backward_ad=False, check_undefined_grad=False,
                          check_batched_grad=False)
 
+######################################################################
+# Functional API
+# --------------------------------------------------------------------
+# We also offer a higher-level functional API in functorch
+# for computing Jacobian-vector products that you may find simpler to use
+# depending on your use case.
+#
+# The benefit of the functional API is that there isn't a need to understand
+# or use the lower-level dual tensor API and that you can compose it with
+# other functorch transforms (like vmap); the downside is that it offers you
```

Comment:
I suppose that will be covered in the advanced tutorial? Maybe we want to link to that here, once that is ready.

Reply:
Compositions of transforms will be covered in the other tutorial (or tutorials) that I am writing, I'm not sure how many. I can link to them when they're done